app/.storybook/main.js

```diff
@@ -7,6 +7,19 @@
 // Support deploying to a subdirectory, such as GitHub Pages.
 const NEXT_PUBLIC_BASE_PATH = process.env.NEXT_PUBLIC_BASE_PATH ?? "";
 
+/**
+ * Most projects use Storybook for internal purposes, and partners may be
+ * sensitive to the Storybook showing up in Google results.
+ * @param {string} head
+ * @returns string
+ */
+function blockSearchEnginesInHead(head) {
+  return `
+    ${head}
+    <meta name="robots" content="none">
```

Comment:
more for my own curiosity, is it stylistically preferrable to do self-closing tag like `<meta .../>` or is it better to be concise like `<meta>`

Reply:
Timely question, you just reminded me of this [recent spicy post](https://jakearchibald.com/2023/against-self-closing-tags-in-html/) that argued against self-closing tags since they're meaningless in HTML and are an artifact from XML. Pretty divisive topic it seems 😅 I might switch this to self-closing to be consistent with how Prettier formats tags though.